dist/default-build-script/Assets/Editor/UnityBuilderAction/Builder.cs

```diff
@@ -39,12 +39,14 @@ public static void BuildProject()
 
       // Set version for this build
       VersionApplicator.SetVersion(options["buildVersion"]);
-      VersionApplicator.SetAndroidVersionCode(options["androidVersionCode"]);
       
       // Apply Android settings
       if (buildPlayerOptions.target == BuildTarget.Android)
+      {
+        VersionApplicator.SetAndroidVersionCode(options["androidVersionCode"]);
```

Comment:
@davidmfinol does this look correct to you too?

Reply:
Yep! I think it makes sense to move this line so it's only applied when it's actually needed.